dataprofiler/profilers/helpers/report_helpers.py

```diff
@@ -87,6 +87,24 @@ def _prepare_report(report, output_format=None, omit_keys=None):
             "data_stats.*.statistics.histogram"
         ])
         output_format = "pretty"
+
+    # Modify omit_keys to account for data_stats being a list and not a dict
+    # With column names for keys
+    new_omit_keys = []
+    for omit_key in omit_keys:
+        key_list = omit_key.split(".")
+        if len(key_list) > 1 and key_list[0] == "data_stats" \
+                and key_list[1] != "*":
+            idxs = report["global_stats"]["profile_schema"][key_list[1]]
+            for i in idxs:
+                swapped_list = key_list
+                swapped_list[1] = str(i)
+                swapped_str = ".".join(swapped_list)
+                new_omit_keys.append(swapped_str)
+        else:
+            new_omit_keys.append(omit_key)
+
+    omit_keys = new_omit_keys
```

Comment:
do we need to restructured this? can we keep it as a list of strings?

Instead when we are at data_stats, can we knowingly if structured match what could be * or otherwise to the column_name?

Reply:
Personally, I wonder if _prepare report would be better if it kept a path and utilized regex of the omits.